docs/quickstart.md

```diff
@@ -67,7 +67,36 @@ Data can be split in sets implementing custom splits according to the specific a
 Assuming that the training, validation and testing ids have been chosen (keys of the hdf5 file), then the corresponding graphs can be saved in hdf5 files containing only references (external links) to the original one. For example:
 
 ```python
-from deeprankcore.DataSet import save_hdf5_keys
+from deeprankcore.NeuralNet import NeuralNet
```

Comment:
I think that `from deeprankcore.DataSet import save_hdf5_keys` should stay and lines from 70 to 99 should be deleted (see below, HDF5DataSet and NeuralNet objects are initialized there)